guides/v2.3/rest/modules/inventory/manage-sources.md

```diff
@@ -0,0 +1,246 @@
+---
+group: rest
+title: Manage sources
+---
+
+Sources represent locations storing and shipping available product stock. Any location with available stock and capable of order fulfillment can be added as a source. These locations can include warehouses, brick-and-mortar stores, distribution centers, and drop shippers.
+
+All stores begin with a default source that must remain enabled. Single Source merchants (merchants who ship all products from one location) use the default source for their single point of inventory location and shipments. Multi Source merchants create as many sources as they need to represent each location.
+
+You cannot rename, delete, or disable the default source. You can create, modify, enable, and disable custom sources, but you cannot rename or delete a custom source.
```

Comment:
Might be worth noting for single source merchants, we recommend updating point of contact and address for the default source?  Since they may use the API to edit that? Eh, not a hot add but mentioning this info in case.